### PR TITLE
this program is to find simpleintrest.

### PR DESCRIPTION
**Pull request by shivamnishad246**
### **Describe your change:**

* [x] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**

* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Java files are placed inside an existing directory.
* [x] All filenames are in all uppercase characters with no spaces or dashes.
* [x] All functions and variable names follow Java naming conventions.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Find simpleintrest [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+		//int p=200;
+		//int r=2;
+		//int t=12;
+		int SI=(p*r*t)/100;
+		System.out.println("simpe interest" + SI);
+		//System.out.println(SI);
+
+	}
+
+}
```
Comment (shivamnew) [inline Find simpleintrest:20]: nice work

**Review by shivamnew — CHANGES_REQUESTED** [review]

**Review by shivamnew — APPROVED** [review]

**Comment by netra2201** [comment]
can you assign this task to me

**Comment by shivamnew** [comment]
yahh sure . I will assign to you.

**Comment by shivamnew** [comment]
HI Netra ,
I have pulled the request in the given repository .named as
Hacktoberfest2020-MovieDo
https://github.com/netra2201/Hacktoberfest2020-MovieDo

thanks

**Pull request closed without merging**